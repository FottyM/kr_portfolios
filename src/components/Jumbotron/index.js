import React from 'react'
import { Link } from 'react-router-dom'

const Jumbotron = () => {
  return (
    <div className="jumbotron">
      <h1 className="display-4">My Wallet!</h1>
      <p className="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.</p>
      <hr className="my-4" />
      <p>It uses utility  for typography and spacing to space content out within the larger container.</p>
      <Link to='/login' className="btn btn-primary btn-lg" href="#" role="button"> Login </Link>
      <Link to='/register' className="btn btn-success btn-lg" href="#" role="button"> Register </Link>
    </div>
  )
}

export default Jumbotron