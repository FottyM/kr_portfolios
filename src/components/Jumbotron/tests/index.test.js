import React from 'react'
import { shallow } from 'enzyme'
import Jumbotron from '../index'

describe('<Jumbotro />', () => {
  it('renders', () => {
    const component = shallow(<Jumbotron />)
    expect(component).toMatchSnapshot()
  })
})
