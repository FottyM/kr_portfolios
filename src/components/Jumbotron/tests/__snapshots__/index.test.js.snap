// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Jumbotro /> renders 1`] = `
<div
  className="jumbotron"
>
  <h1
    className="display-4"
  >
    My Wallet!
  </h1>
  <p
    className="lead"
  >
    This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.
  </p>
  <hr
    className="my-4"
  />
  <p>
    It uses utility  for typography and spacing to space content out within the larger container.
  </p>
  <Link
    className="btn btn-primary btn-lg"
    href="#"
    replace={false}
    role="button"
    to="/login"
  >
     Login 
  </Link>
  <Link
    className="btn btn-success btn-lg"
    href="#"
    replace={false}
    role="button"
    to="/register"
  >
     Register 
  </Link>
</div>
`;
